client/js/app.jsx

```diff
@@ -29,7 +29,7 @@ class Root extends React.Component {
     const { store } = this.props;
     return (
       <Provider store={store}>
-        <div>
+        <div style={{"height": "100%"}}>
```

Comment:
We need the style there to be 100% or else everything goes wonky. I'm not even sure what stylesheet we would use to target that